### PR TITLE
Build and release using Github actions (close #27)

### LINKED ISSUE #27: Build and release using Github actions

### PR DESCRIPTION
**Pull request by matus-tomlein**
This was already reviewed in PR #29 but I added a Github action to build and run tests under Windows and so I decided reopen this branch.

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      uses: actions/checkout@v2
+
+    - name: Setup MSBuild and add to PATH
+      uses: microsoft/setup-msbuild@v1.1
+
+    - name: Run MSBuild
+      run: msbuild .\snowplow-cpp-tracker.sln
+
+    - name: Run tests
+      run: .\x64\Debug\snowplow-cpp-tracker.exe
```
Comment (paulboocock) [inline .github/workflows/build.yml:33]: I haven't tested this, but does running the .exe really run the tests? That seems wild if true.

Comment (matus-tomlein) [reply .github/workflows/build.yml:33]: Yup, it does. The Visual Studio solution file just builds an executable to run the unit tests. I was really happy to discover this although it took a few hours of pain in virtual machines.

Screenshot 2022-02-10 at 16 51 57

**Review by AlexBenny — APPROVED** [review]
Awesome!

**Pull request merged**